build/lib/i18n.ts

```diff
@@ -904,6 +905,27 @@ export function createI18nFile(name: string, messages: any): File { // {{SQL CAR
 		'Do not edit this file. It is machine generated.'
 	];
 	for (const key of Object.keys(messages)) {
+		// {{SQL CARBON EDIT}} - Need to handle aliased ids from vscode-translations-export
+		let areaList = messages['contents'];
+		if (name.startsWith('extensions/') && key.startsWith('contents') && areaList) {
+			for (const areaKey of Object.keys(areaList)) {
+				let stringList = messages['contents'][areaKey];
+				for (const stringKey of Object.keys(stringList)) {
+					let shortName = name.substring(11);
```

Comment:
where does 11 come from? Please add a comment. Same for below when getting the `substring(8)`